pulsar/managers/util/drmaa/__init__.py

```diff
@@ -41,13 +41,14 @@ class DrmaaSession(object):
 
     def __init__(self, session_constructor, **kwds):
         with self._session_lock():
+            DrmaaSession.session_count += 1
```

Comment:
Well clearly this coding thing isn't working out for me today - this would be a good day for that Pittsburgh trip.